types/mongodb/index.d.ts

```diff
@@ -1464,11 +1464,11 @@ export interface DeleteWriteOpResultObject {
         ok?: number;
         //The total count of documents deleted.
         n?: number;
+        //The number of documents deleted.
+        deletedCount?: number;
```

Comment:
I think the old version was correct. According to the documentation `deletedCount` should be one level higher:
http://mongodb.github.io/node-mongodb-native/3.1/api/lib_collection.js.html#line842

Reply:
indeed. i've posted the log from mongodb here https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37501#issuecomment-521194331